### PR TITLE
[TASK] Implement global sequence number (DC-90)

### PR DESCRIPTION
**Pull request by mrajvanshy**

### CODE REVIEW

File: src/js/controllers/app.js [hunk 1]
```diff
@@ -19,6 +19,21 @@ module.controller('AppCtrl', ['$rootScope', '$compile', 'rpId', 'rpNetwork',
 
   var account;
 
+  // Global sequence variable to be incremented after every transaction
+  $scope.$watchGroup(['userBlob', 'sequence'], function() {
```
Comment (vhpoet) [inline src/js/controllers/app.js]: why are you listening to `sequence` here?

File: src/js/tabs/send.js [hunk 2]
```diff
@@ -792,7 +792,8 @@ SendTab.prototype.angular = function (module)
         tx.submit();
       }
       else {
-        tx.tx_json.Sequence = Number($scope.send.sequence);
+        tx.tx_json.Sequence = Number($scope.sequence);
+        $scope.incrementSequence($scope.sequence);
```
Comment (vhpoet) [inline src/js/tabs/send.js]: You don't need to pass the `$scope.sequence` to the `incrementSequence` function. It already has access to the `$scope.sequence`.

**Comment by vhpoet** [comment]
Does changing the sequence number in any of controllers propagate the change to the root scope, or does it stay local?

**Comment by mrajvanshy** [comment]
@vhpoet it propagates the change to the root scope.

**Comment by vhpoet** [comment]
LGTM

**Pull request merged**